### PR TITLE
Fix login with amazon and integrate paypal

### PR DESCRIPTION
**Pull request by nitin007**

### CODE REVIEW

**Comment by nitin007** [comment]
@jitendra Vinsol's fork is behind spree-contrib repo. So, you will be seeing commits from other authors also.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Contributing
 
+Spree Social is an open source project and we encourage contributions. Please see the [contributors guidelines](http://spreecommerce.com/documentation/contributing_to_spree.html) for more information before contributing.
```
Comment (nishant-cyro) [inline CONTRIBUTING.md:3]: @nitin007 - :+1:

**Comment by nishant-cyro** [comment]
@nitin007 - Can you close this one and create a new PR.

As this fork is not updated earlier and has too many not required commits.

**Comment by nitin007** [comment]
Sure

**Pull request closed without merging**